test/mocha/helpers.js

```diff
@@ -31,7 +33,12 @@ const credentialsEdv = 'credentials-edv-document';
 
 //ensure each test has a fresh credential
 function cloneCredential() {
-  return JSON.parse(JSON.stringify(credential));
+  return util.clone(credential);
+}
+
+//ensure each test has a fresh credential
+function cloneAuthPresentation() {
+  return util.clone(authPresentation);
```

Comment:
I'm thinking that this style of helper might not be the most useful.  The thing I'm noticing is that they are stuck with just a single credential, or a single presentation.  it's often the case that we want to use multiple mocks in a test suite.  So the  enhancement would be that a credential/presentation would be passed in, but if that were going to happen, then we're back to just using `util.clone` in the test spec.  I think using `util.clone` in the test spec is probably a good idea because it gives the spec writer visibility and control over the mock data.  As is, someone would have to root around in the code to determine what `cloneCredential` actually does.

Other than that, thanks as always for working on tests, and things are headed in the right direction.

Reply:
Yeah I do agree with you here so I consolidated both the json test data objects into ` mockData.json` file and now call clone on them in the test suite: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/49/commits/647734731b8980c8d4a4be6360d4afb2a187f5af

It is definitely better than having 2 separate functions.